### PR TITLE
Updated DEVELOP.md to pywbemtools project

### PR DESCRIPTION
**Pull request by andy-maier**
Ready for review and merge.

### CODE REVIEW

File: DEVELOP.md [hunk 1]
```diff
@@ -84,63 +82,30 @@ Their upstream repos are assumed to have the remote name `origin`.
       version. In the "Starting a new version" section, the link will be added
       again for the new version.
 
-5.  Edit the README file:
```
Comment (KSchopmeyer) [inline DEVELOP.md]: Actually, those variables are in the README as I have it in another PR.  Shouldn't we be keeping the original item 5?

File: DEVELOP.md [hunk 2]
```diff
@@ -153,48 +118,29 @@ Their upstream repos are assumed to have the remote name `origin`.
     the `master` branch. If you are releasing a fix version, you need to
     change the target branch of the Pull Request to `stable_M.N`.
 
-10. Optional: Perform a complete test using Tox:
```
Comment (KSchopmeyer) [inline DEVELOP.md]: Morequestion than anything else.  Why did you remove these test steps???

File: DEVELOP.md [hunk 3]
```diff
@@ -204,18 +150,18 @@ Their upstream repos are assumed to have the remote name `origin`.
...
     will cause it to appear in the Release tab.
 
-19. On GitHub, close milestone `M.N.U`.
+15. On GitHub, close milestone `M.N.U`.
 
     Note: Issues with that milestone will be moved forward in the section
     "Starting a new version".
 
-20. Upload the package to PyPI:
+16. Upload the package to PyPI:
```
Comment (KSchopmeyer) [inline DEVELOP.md]: Question. Is there a means to test upload to PyPi?

Second, does that mean that if there is an issue with the upload we have to go to new minor version?

File: DEVELOP.md (hunk 1 above)
Comment (andy-maier) [reply DEVELOP.md]: We agreed that we would no longer have the versons etc in the README file, so this one is resolved.

File: DEVELOP.md (hunk 3 above)
Comment (andy-maier) [reply DEVELOP.md]: Testing pypi uload is not possible, to my knowledge.
Yes.
